From 4f6c0bc66e13f9766e2042054e6ce1a1a3730907 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:05:07 -0400
Subject: [tools] perf/symbols: No need to export dso__first_symbol

Message-id: <1410181752-25631-63-git-send-email-jolsa@redhat.com>
Patchwork-id: 91741
O-Subject: [PATCH RHEL7.1 BZ1134356 062/307] perf symbols: No need to export dso__first_symbol
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit c96626b1da589075b1b3e815239ceace11350662
Author: Arnaldo Carvalho de Melo <acme@redhat.com>
Date: Fri Feb 14 17:09:10 2014 -0300

description
===========
There are no users outside the file that defines it.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/util/symbol.c b/tools/perf/util/symbol.c
index e89afc0..c3b0147 100644
--- a/tools/perf/util/symbol.c
+++ b/tools/perf/util/symbol.c
@@ -410,7 +410,7 @@ struct symbol *dso__find_symbol(struct dso *dso,
  return symbols__find(&dso->symbols[type], addr);
 }
 
-struct symbol *dso__first_symbol(struct dso *dso, enum map_type type)
+static struct symbol *dso__first_symbol(struct dso *dso, enum map_type type)
 {
  return symbols__first(&dso->symbols[type]);
 }
diff --git a/tools/perf/util/symbol.h b/tools/perf/util/symbol.h
index fffe288..538d484 100644
--- a/tools/perf/util/symbol.h
+++ b/tools/perf/util/symbol.h
@@ -223,7 +223,6 @@ struct symbol *dso__find_symbol(struct dso *dso, enum map_type type,
     u64 addr);
 struct symbol *dso__find_symbol_by_name(struct dso *dso, enum map_type type,
      const char *name);
-struct symbol *dso__first_symbol(struct dso *dso, enum map_type type);
 
 int filename__read_build_id(const char *filename, void *bf, size_t size);
 int sysfs__read_build_id(const char *filename, void *bf, size_t size);
-- 
1.7.1